daprdocs/content/en/python-sdk-docs/python-workflow.md

```diff
@@ -0,0 +1,146 @@
+- The workflow activity definitions are found in the `todo` directory.
+
+## Prerequisites
+- [Dapr CLI]({{< ref install-dapr-cli.md >}}) installed
+- Initialized [Dapr environment]({{< ref install-dapr-selfhost.md >}})
+- [Python 3.7+](https://www.python.org/downloads/) installed
+- [Dapr Python module]({{< ref "python#install-the0dapr-module" >}}) installed
+- Verify you're using the latest proto bindings
+
+## Set up the environment
+
+If it's not already installed, run the following command to install the Dapr Python SDK.
+
+```bash
+pip3 install dapr
+```
+
+Clone the [Python SDK repo].
+
+```bash
+git clone https://github.com/dapr/python-sdk.git
+```
+
+From the Python SDK root directory, navigate to the Dapr Workflow example.
+
+```bash
+cd examples/workflow
+```
+
+## Run the application locally
+
+To run the Dapr application, you need to start the Python program and a Dapr sidecar. In the terminal, start the sidecar.
+
+```bash
+dapr run -- python3 workflow.py
+```
+
+> **Note:** Since Python3.exe is not defined in Windows, you may need to use `python workflow.py` instead of `python3 workflow.py`.
+
+
+## Start a workflow
+
+To start a workflow, you have two options:
+
+1. Follow the directions from the console prompts.
+1. Use the workflow API and send a request to Dapr directly. 
+
+This guide focuses on the workflow API option. 
+
+{{% alert title="Note" color="primary" %}}
+  - You can find the commands below in the `todo` file.
+  - The body of the curl request is the purchase order information used as the input of the workflow. 
+  - The "1234" in the commands represents the unique identifier for the workflow and can be replaced with any identifier of your choosing.
+{{% /alert %}}
+
+Run the following command to start a workflow. 
+
+{{< tabs "Linux/MacOS" "Windows">}}
+
+{{% codetab %}}
+
+```bash
+todo
+```
+
+{{% /codetab %}}
+
+{{% codetab %}}
+
+```powershell
+todo
+```
+
+{{% /codetab %}}
+
+{{< /tabs >}}
+
+If successful, you should see a response like the following: 
+
+```json
+todo
+```
+
+Send an HTTP request to get the status of the workflow that was started:
+
+```bash
+todo
+```
+
+The workflow is designed to take several seconds to complete. If the workflow hasn't completed when you issue the HTTP request, you'll see the following JSON response (formatted for readability) with workflow status as `RUNNING`:
+
+```json
+todo
+```
+
+Once the workflow has completed running, you should see the following output, indicating that it has reached the `COMPLETED` status:
+
+```json
+todo
+```
+
+When the workflow has completed, the stdout of the workflow app should look like:
+
+```log
+todo
+```
+
+
+## Error Handling
+
+The Dapr Python SDK will pass through errors that it receives from the Dapr runtime. In the case of an ETag mismatch, the Dapr runtime will return `StatusCode.ABORTED`.
+
+**Example**
+
+```
+== APP == State store has successfully saved value_1 with key_1 as key
```

Comment:
These outputs are not currently correct and are only present in the python PR since the output at this time is not known. Once the example app is created the expected output will be updated